docs/modules/ROOT/pages/how-tos/configuring/prefetching-dependencies.adoc

```diff
@@ -204,6 +204,60 @@ spec:
 * From the {ProductName} *Applications* view, go to *Activity > Latest commits*.
 //include::../partials/con_hermetic_verification.adoc[]
 
+=== Prefetching `pip` dependencies from custom index servers
+
+Cachi2 supports the link:https://pip.pypa.io/en/stable/cli/pip_install/#install-index-url[--index-url] option.
+You can add this option to your `requirements.txt` file(s), instructing Cachi2 to download packages from the specified
+index server. Your file may look something like this:
+
+[source,text]
+----
+--index-url=https://example.pypi.org/simple/
+requests==2.32.2 \
+    --hash=sha256:dd951ff5ecf3e3b3aa26b40703ba77495dab41da839ae72ef3c8e5d8e2433289 \
+    --hash=sha256:fc06670dd0ed212426dfeb94fc1b983d917c4f9847c863f313c9dfaaffb7c23c
+
+# ...other packages
+----
+
+WARNING: *Do not include credentials in the index url.* If needed, provide authentication via a `.netrc` file (as described below).
+
+To provide `.netrc` authentication for the index server:
+
+. Create a key/value secret (see xref:/how-tos/configuring/creating-secrets.adoc[creating secrets for your builds]).
+  Set `.netrc` as the key. Upload the content of your `.netrc` file as the value. For more details on netrc files,
+  see https://pip.pypa.io/en/stable/topics/authentication/#netrc-support.
+
+. In your `.tekton/` PipelineRun files, pass the netrc secret to the pipeline. Add the newly created secret to the `.spec.workspaces` section:
```

Comment:
```suggestion
. In your `.tekton/` PipelineRun files, pass the netrc secret to the pipeline. 
. Add the newly created secret to the `.spec.workspaces` section:
```

Reply:
It is one step - the user passes the netrc secret to the pipeline by adding it to `.spec.workspaces`

Reply:
I dropped `pass the netrc secret to the pipeline.`. It was meant to explain what the user achieves by adding the workspace, but I think it only added confusion